src/utils/atomic.h

```diff
@@ -61,6 +61,10 @@ struct atomic_t {
 
 #if 0
 
+#if _BullseyeCoverage
```

Comment:
it is undef `#if 0`

Reply:
I know, but this pragma is missing - see line 92